corehq/apps/smsforms/util.py

```diff
@@ -19,6 +19,5 @@ def process_sms_form_complete(session, form, completed):
     session.save()
 
     if not completed:
-        xform.partial_submission = True
```

Comment:
Could be worth adding this as an optional arg to `SubmisionPost` to avoid the duplicate save and extra SQL function.